pages/project-version/middleware/index.js

```diff
@@ -110,8 +113,51 @@ const canComment = () => (req, res, next) => {
     .catch(next);
 };
 
+const getPreviousVersion = () => (req, res, next) => {
+
+  const versions = orderBy(req.project.versions.filter(pv => pv.id !== req.versionId),
+    v => moment(v.createdAt).format('YYYY-MM-DD'), ['desc']);
+
+  if (versions.length > 0) {
+    let prevVersion = versions.find(v => {
+      if (moment(req.version.createdAt).isAfter(v.createdAt)) {
```

Comment:
you don't need an `if` here - just return the condition

Reply:
done